src/utils/authFetch.js

```diff
@@ -0,0 +1,19 @@
+import {jwtUtils} from "./jwt/jwtUtils";
+import {loginAPI} from "./apis/LoginAPI";
+
+export const authFetch = (url, data) => {
+    const authToken = jwtUtils.getAuthToken()
+    const dataHeaders = data?.headers
+    const authHeaders = authToken ? {[loginAPI.jwtTokenKey]: authToken} : {}
+    const newHeaders = dataHeaders ? Object.assign(dataHeaders, authHeaders) : authHeaders
+    const authData = Object.assign(data | {}, {headers: newHeaders})
+    return fetch(url, authData).then(response => {
+        if(response.status >= 200 && response.status < 300)
+            return response
```

Comment:
Znaczy to jest może głupie pytanie ale returna nie wystarczyłoby dać przed if-a?

Reply:
Ale że jak, bo nwm jak by to miało wyglądać xd

Reply:
return przed ifem jest w kotlinie, tu może go nie być XD